### ISSUE TITLE
Where is it after I close it.

### ISSUE THREAD

**Issue by shannon247** [MicrosoftDocs/Virtualization-Documentation#1016]
I created a Linux machine. However, now that I've closed it, where is it?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 10344c5d-274e-de75-ecb6-11d53a377986
* Version Independent ID: 4cea53b8-1864-f12b-f0d9-f208743475c8
* Content: [Create a Virtual Machine with Hyper-V](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/quick-create-virtual-machine#feedback)
* Content Source: [virtualization/hyper-v-on-windows/quick-start/quick-create-virtual-machine.md](https://github.com/Microsoft/Virtualization-Documentation/blob/master/virtualization/hyper-v-on-windows/quick-start/quick-create-virtual-machine.md)
* Product: **windows-10-hyperv**
* GitHub Login: @scooley

**Comment by MythreyaK** [comment]
It should show up in the `Hyper-V Manager` (sidebar on the left, under your hostname)

![image](https://user-images.githubusercontent.com/26112391/54517289-4e9f8300-4987-11e9-93c4-cc63c3dcd43f.png)

**Comment by metablaster** [comment]
Indeed, just make sure you run Hyper-V manager as administrator otherwise you won't see anything there.

**Comment by robinharwood** [comment]
We hugely appreciate your contribution, feedback, and support to our technical documentation.

We’ve phased out GitHub Issues on this repository as the feedback mechanism for content and have replaced it with a new feedback system. For more information see, [Content User Feedback](https://aka.ms/ContentUserFeedback). As part of this rollout, we are migrating your issue where our team will use your comments to improve the documentation experience. Unfortunately, this does mean you won't be able to access this issue after the migration is complete.

All issues will be migrated by Thursday 29th August 2024.

Once again, thank you for your valuable feedback, we hope you’ll continue to contribute using our new feedback system.
